static/js/context.js

```diff
@@ -1,14 +1,320 @@
-import React from 'react';
+import React, { useContext, useEffect, useState } from "react";
+
 const ContentLanguageContext = React.createContext({
   language: "english",
 });
-ContentLanguageContext.displayName = 'ContentLanguageContext'; //This lets us see this name in the devtools
+ContentLanguageContext.displayName = "ContentLanguageContext"; //This lets us see this name in the devtools
 
-const AdContext = React.createContext({
-});
-AdContext.displayName = 'AdContext';
+const AdContext = React.createContext({});
+AdContext.displayName = "AdContext";
+
+const StrapiDataContext = React.createContext({});
+StrapiDataContext.displayName = "StrapiDataContext";
+
+const transformValues = (obj, callback) => {
```

Comment:
It's meant to be a utility function but I didn't see something similar elsewhere. It could be shared. I think it's a common pattern. Should all utility functions go in sefaria.js?